### DISCUSSION TITLE
Darkly Theme on mobile has display issues

### DISCUSSION THREAD

**Discussion by CBB3810** [pglombardo/PasswordPusher#1258] [Q&A] [answered]
![Securely Send a Password  Credential Mgmt](https://github.com/pglombardo/PasswordPusher/assets/123845047/e9ca5326-905d-4b04-a88e-afcda29fbcbb)

See the **anomaly** in the text box. Can anyone else confirm this?

**Comment by pglombardo** [comment, marked as answer]
Hi @CBB3810 - confirmed.  The password delivery page also doesn't display correctly since the password and background are both white.  These are unfortunate side affects with these themes taken directly from the bootswatch project.

I created #1267 to track.  I'm not sure when and how I'll fix this (without changing the packaged themes) but in the meantime you could [add custom CSS overrides](https://github.com/pglombardo/PasswordPusher/blob/master/Configuration.md#how-to-add-custom-css) to fix.

The following custom CSS would likely fix these issues:

```css
#the-count {
  color: black;
}

#push-payload {
  color: black;
}
```
